otx/algorithms/anomaly/adapters/anomalib/data/data.py

```diff
@@ -93,23 +94,44 @@ def __getitem__(self, index: int) -> Dict[str, Union[int, Tensor]]:
         dataset_item = self.dataset[index]
         item: Dict[str, Union[int, Tensor]] = {}
         item = {"index": index}
-        if self.task_type in (TaskType.ANOMALY_CLASSIFICATION, TaskType.ANOMALY_DETECTION):
+        if self.task_type == TaskType.ANOMALY_CLASSIFICATION:
             # Detection currently relies on image labels only, meaning it'll use image
             #   threshold to find the predicted bounding boxes.
-            item["image"] = self.pre_processor(image=dataset_item.numpy)["image"]
+            item["image"] = self.transform(image=dataset_item.numpy)["image"]
+        elif self.task_type == TaskType.ANOMALY_DETECTION:
+            item["image"] = self.transform(image=dataset_item.numpy)["image"]
+            item["boxes"] = torch.empty((0, 4))
+            height, width = self.config.dataset.image_size
+            boxes = []
+            for annotation in dataset_item.get_annotations():
+                if not Rectangle.is_full_box(annotation.shape):  # ignore full image labels
+                    boxes.append(
+                        Tensor(
+                            [
+                                annotation.shape.x1 * width,
+                                annotation.shape.y1 * height,
+                                annotation.shape.x2 * width,
+                                annotation.shape.y2 * height,
+                            ]
+                        )
+                    )
+                if len(boxes) > 0:
```

Comment:
A suggestion by `refurb`
```suggestion
                if boxes:
```